scripts/docker-develop/Dockerfile

```diff
@@ -13,9 +13,6 @@ FROM openjdk:8u171
 ENV SCALA_VERSION 2.12.9
 ENV SBT_VERSION 1.2.8
 
-# Scala expects this file
-RUN touch /usr/lib/jvm/java-8-openjdk-amd64/release
```

Comment:
No longer expected?

Reply:
Apparently not, and I couldn't find anything resembling that directory path anyway so that's fortunate.